circle.yml

```diff
@@ -11,6 +11,10 @@ general:
     - http-servers/build/libs
     - ssl-config/build/libs
 
+dependencies:
+  override:
+    - ./gradlew resolveDependencies
```

Comment:
Not quite, for a few reasons (also, I couldn't find `assembleTest` as a task, so assume you're referring to `assemble`).

First off, it actually doesn't download all the dependencies that `resolveDependencies` does. Compare the output of the `build` step using `resolveDependencies` (https://circleci.com/gh/palantir/http-remoting/135) versus `assemble` (https://circleci.com/gh/palantir/http-remoting/136) -- the former properly grabs all dependencies so that the `build` step doesn't need to download anything further, while the latter does not.

RAINBOW has documented this issue further and suggests this work-around -- see issue #21 in RAINBOW's "build-system-issues" repository in GHE for more info.